easybuild/easyconfigs/i/inferCNV/inferCNV-1.16.0-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'Bundle'
+
+name = 'inferCNV'
+version = '1.16.0'
+versionsuffix = '-R-%(rver)s'
+local_biocver = '3.15'
+
+homepage = 'https://github.com/broadinstitute/inferCNV/wiki'
+description = """InferCNV is used to explore tumor single cell RNA-Seq data to identify evidence
+ for somatic large-scale chromosomal copy number alterations, such as gains or
+ deletions of entire chromosomes or large segments of chromosomes."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('R', '4.2.1'),
+    ('R-bundle-Bioconductor', local_biocver, '-R-%(rver)s'),
+    ('rjags', '4-13', '-R-%(rver)s'),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://bioconductor.org/packages/release/bioc/src/contrib/',  # current version of packages
```

Comment:
Sorry, I made a mistake with my previous comment as I did not realise this was coming from Bioconductor (instead of their github).

inferCNV 1.16.0 is with Bioconductor 3.17: https://www.bioconductor.org/packages/3.17/bioc/html/infercnv.html

The inferCNV with Bioconductor 3.15 was 1.12.0, see https://www.bioconductor.org/packages/3.15/bioc/html/infercnv.html

Reply:
Oh, I see. So I will update the `R-bundle-Bioconductor` easyconfig.

Reply:
Oh, the issue says "must be compatible with R-bundle-Bioconductor/3.15-foss-2022a-R-4.2.1" so I will use the 1.12.0 version of `inferCNV`.